pkg/apis/jenkins.io/v1/types_pipeline.go

```diff
@@ -71,10 +71,11 @@ type BatchPipelineActivity struct {
 	ComprisingPulLRequests []PullRequestInfo `json:"pullRequestInfo,omitempty" protobuf:"bytes,3,opt,name=pullRequestInfo"`
 }
 
-// PullRequestInfo contains information about a PR, like its PR and Build numbers
+// PullRequestInfo contains information about a PR, like its PR, Build numbers, and SHA
 type PullRequestInfo struct {
 	PullRequestNumber        string `json:"pullRequestNumber,omitempty" protobuf:"bytes,1,opt,name=pullRequestNumber"`
 	LastBuildNumberForCommit string `json:"lastBuildNumberForCommit,omitempty" protobuf:"bytes,2,opt,name=lastBuildNumberForCommit"`
+	LastBuildSHA             string `json:"lastBuildSHA,omitempty" protobuf:"bytes,3,opt,name=lastBuildSHA"`
```

Comment:
What's about adding some GoDoc to _LastBuildSHA _ and _LastBuildNumberForCommit_? _PullRequestNumber_ seems obvious, the other two not so much. At least not to me atm.

Reply:
Will do!

Reply:
thanks